sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -74,7 +74,7 @@
   "autoPublish": false,
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
-    "@azure/core-auth": "^1.1.3",
+    "@azure/core-auth": "^1.1.4",
```

Comment:
Does updating only core-auth but not core-http make sense?

Reply:
I remember we wanted to bump the core-http version after releasing but I don't see why we couldn't do it here.  Will update.

Reply:
On a second though, that might trigger review request for many non-core libraries. I prefer to do it later.

Reply:
My concern is on being in a state where the core-auth and core-http versions are using different pinned versions of core-tracing. 

> On a second though, that might trigger review request for many non-core libraries. I prefer to do it later.

Then, I would recommend closing this PR, and include these changes in the PR that you were planning to create for updating core-http

Reply:
We also want to release core-http that depends on new version of core-auth.  I was thinking of releasing both of them in the same release pipeline thus this PR.  I can do separate release pipeline then.